browser/src/UI/components/QuickInfoContainer.tsx

```diff
@@ -0,0 +1,166 @@
+import { merge } from "lodash"
+import * as React from "react"
+
+import { IThemeColors, ThemeProvider, withTheme } from "./common"
+import { QuickInfoContainer, QuickInfoDocumentation, QuickInfoTitle } from "./QuickInfo"
+
+interface IQuickInfoProps {
+    theme: IThemeColors
+    titleAndContents: ITitleAndContents
+}
+
+interface ITitleAndContents {
+    title: {
+        __html: string
+    }
+    description: {
+        __html: string
+    }
+}
+
+const quickInfoTokens = {
+    "editor.tokenColors": {
+        "meta.import": {
+            scope: "meta-import",
+            settings: "Define",
```

Comment:
Ah ya, this is where I was hoping we could specify color/background color/bold/italic etc (and in the user's config / theme as well) instead of the vim highlight group. The vim highlight group was really just a crutch to implement.